app/src/main/java/com/willowtree/vocable/keyboard/KeyboardViewModel.kt

```diff
@@ -16,23 +21,43 @@ class KeyboardViewModel : BaseViewModel() {
     }
 
     private val presetsRepository: PresetsRepository by inject()
+    private val mySayingsCategoryId: String =
+        get<Context>().getString(R.string.category_my_sayings_id)
```

Comment:
I'm not familiar with this syntax `get<Context>()`.  Is this just the App Context from your Koin module?

Reply:
Yes I think Koin allows us to have a context in the view model

Reply:
Also, It might be better to have this just return the resource id and get the string in the view, rather than accessing the context in the viewmodel, even if it is being done through your DI.

Reply:
Moved passing this in via a view model factory